integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -13,7 +13,16 @@
 # limitations under the License.
 
 import pytest
-import pandas as pd
+
+pd = pytest.importorskip("pandas", reason="pandas is required for pandas UDF testing")
```

Comment:
we do not need this for cudf_udf tests if [PR 824](https://github.com/NVIDIA/spark-rapids/pull/824) is merged. Besides pandas only check is not enough for cudf_udf tests.

Reply:
I don't see how having extra checks is a problem.  Having cudf_udf off by default does not change that someone might not have set up the environment properly.

Reply:
the requirements documentation in pr 824 mentions nothing of pandas or pyarrow, do those come with the cudf install?

Reply:
I'm not 100% sure.  I now that arrow does, but I am not sure on pandas.  I think it does.

Reply:
yes, conda will install all the dependencies of cudf when installing cudf, including pandas and pyarrow.

Reply:
>I don't see how having extra checks is a problem. Having cudf_udf off by default does not change that someone might not have set up the environment properly.

make sense to have this extra check. There is a case of installing cudf python from source code, then no pandas or pyarrow installed automatically.